src/Resources/config/admin.xml

```diff
@@ -14,5 +14,18 @@
             <tag name="form.type" alias="sonata_security_roles"/>
             <argument type="service" id="sonata.user.editable_role_builder"/>
         </service>
+        <service id="sonata.user.roles_matrix_builder" class="Sonata\UserBundle\Security\RolesMatrixBuilder">
```

Comment:
please rename and use class as id

Reply:
@OskarStark same question , why would we use the class as id?

Reply:
If I am right, we need an `id`, but not a `class` if `id` can be resolved, and here we are using it this way:
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/config/core.xml#L14

In yaml I am using it this way:
```yaml
services:
    Nexy\Slack\Client:
        arguments: ['https://hooks.slack.com/services/.....']
```